python/lsst/ts/imsim/closed_loop_task.py

```diff
@@ -838,16 +813,14 @@ def run_wep(
             estimation pipeline for each sensor.
         """
 
-        if inst_name in ["lsst", "lsstfam"]:
+        if cam_type in [CamType.LsstCam, CamType.LsstFamCam]:
```

Comment:
It seems that these two if statements are only used in the if statements below where we run the pipeline, right? Then maybe we could have them together?

Reply:
No, because then we need a different combination of cam_types below. But I can combine the LsstFamCam and ComCom instructions below into one since they are identical.

Reply:
Also replacing this with the new function suggested below should cut down on repitition.